src/html/icons.html

```diff
@@ -21,7 +21,7 @@ <h2 class="h2 weight-strong">Single Color</h2>
 </pre>
   <div class="figure-wrapper">
     <figure class="grid-block figure icon-outline">
-      <div class="grid-block col align-center">
+      <div class="grid-block vertical align-center">
```

Comment:
Let's remove `align-center` from these (and the other figures with buttons, so the buttons are full-width like they were before.